cms/templates/js/publish-xblock.underscore

```diff
@@ -75,10 +75,17 @@ var visibleToStaffOnly = visibilityState === 'staff_only';
                 <% } %>
             </p>
         <% } else { %>
-            <p class="copy"><%= gettext("Staff and Students") %></p>
+            <p class="visbility-copy copy"><%= gettext("Staff and Students") %></p>
         <% } %>
-        <p class="action-inline">
-            <a href="" class="action-staff-lock" role="button" aria-pressed="<%= hasExplicitStaffLock %>">
+            <% if (hasContentGroupComponents) { %>
+                <p class="note-visibility">
+                    <i class="icon icon-eye-open"></i>
+                    <span class="note-copy"><%= gettext("Some content in this unit is only visible to particular groups") %></span>
```

Comment:
I'm pretty sure these aren't changes introduced in this PR, but in a previous one that showed up in the diff due to a rebase. @talbs can you confirm?